runtime/experimental/export.Dockerfile

```diff
@@ -20,15 +20,17 @@ RUN rm -rf /opt/bref/data
 
 WORKDIR /opt
 COPY bootstraps/cli.bootstrap /tmp/cli.bootstrap
-RUN export PHP_ZIP_NAME=/export/php-$(php -r '$version = explode(".", phpversion());printf("%d.%d", $version[0], $version[1]);'); \
- mkdir -p /export; \
+RUN export PHP_ZIP_NAME=/export/php-$(php -r '$version = explode(".", phpversion());printf("%d.%d", $version[0], $version[1]);');
+RUN mkdir -p /export; \
  rm -rf /export/*; \
- rm -rf /bootstrap; \
- zip -qr ${PHP_ZIP_NAME}.zip . -x "*php-cgi"; \
```

Comment:
Removed the `php-7.2.zip`: wasn't used afterwards (and it was confusing having that on disk), it just means we zip the files twice but I reckon it doesn't really matter.